caas/kubernetes/provider/namespaces.go

```diff
@@ -123,9 +123,11 @@ func (k *kubernetesClient) ensureNamespaceAnnotations(ns *core.Namespace) error
 // createNamespace creates a named namespace.
 func (k *kubernetesClient) createNamespace(name string) error {
 	ns := &core.Namespace{ObjectMeta: v1.ObjectMeta{Name: name}}
+	ns.ObjectMeta.Labels = LabelsForModel(k.CurrentModel())
```

Comment:
we can do ns.SetLabels(mergeLabels(ns.GetLabels(), LabelsForModel(k.CurrentModel())))

Reply:
Done & Done